### ISSUE TITLE
close icon in filter indicator does not work if filter is created using default()

### ISSUE THREAD

**Issue by flyingeek** [filamentphp/filament#3674]
### Package

filament/tables

### Package Version

v2.15.26

### Laravel Version

v9.26.1

### PHP Version

PHP 8.1.8

### Problem description

When a filter is marked as default, you can deselect the filter from the filter popup but not using the close icon in the indicator.
![Image 2022-08-25 at 3 29 29 PM](https://user-images.githubusercontent.com/68456/186678171-0bb073d3-354a-4d25-a0fe-3e4db2ee5d12.jpg)

### Expected behavior

Two possibilities:

- Make the close button operational
- Remove the close button if the filter is default

### Steps to reproduce

set a Table filter using the default method.

```php
    Tables\Filters\Filter::make('is_featured')->label('Featured')->default(),
```

### Reproduction repository

https://github.com/flyingeek/filament-bug-report

**Comment by danharrin** [comment]
Hey, this was fixed by #3669, but it's not been released yet

**Comment by flyingeek** [comment]
You are too quick !
Anyway I made a nice repo template for my first report. You should link to something similar for reporting bugs in the future.
